getting-started/services.md

```diff
@@ -33,7 +33,7 @@ Now that you have an active $ACCOUNT_LONG, you create and manage your $SERVICE_S
 
 1. In the [$SERVICE_SHORT creation page][create-service], choose the $PG capability or add `Time-series and analytics` and `AI and Vector` capabilities. Click `Save and continue`.
 
-   ![Create a $SERVICE_LONG](https://assets.timescale.com/docs/images/tiger-cloud-console/create-tiger-cloud-service.png)
+   ![Create a $SERVICE_LONG](https://assets.timescale.com/docs/images/tiger-cloud-console/2025-6-20-create-tiger-cloud-service-type.png)
```

Comment:
hi there @solugebefola , you don't need to change the file name. If you add a new version of https://assets.timescale.com/docs/images/tiger-cloud-console/create-tiger-cloud-service.png the image will go live in a few hours as the cache is flushed.